services/test-validators.js

```diff
@@ -155,6 +155,15 @@ const isCustomCompactTestTotals = makeCompactTestTotalsValidator({
   skipped: '🤷',
 })
 
+const isOrdinalNumber = Joi.string().regex(/^[1-9][0-9]*(ᵗʰ|ˢᵗ|ⁿᵈ|ʳᵈ)$/)
+const isOrdinalNumberDaily = Joi.string().regex(
+  /^[1-9][0-9]*(ᵗʰ|ˢᵗ|ⁿᵈ|ʳᵈ) daily$/
+)
+
+const isHumanized = Joi.string().regex(
+  /\d* (second|seconds|minute|minutes|hour|hours|day|days|month|months|year|years)/
```

Comment:
`\d*` will match zero digits. Think we want `\d+` to require one or more.

Reply:
https://day.js.org/docs/en/durations/humanize

In these examples they mention that the result of ‚humanize()’ can sometimes be for example ‚a day’.
I used an asterisk here on purpose :))

![53EEFF2A-1B1F-4173-A137-0F64562795B8](https://user-images.githubusercontent.com/31891675/194726257-2ee8a031-32a5-48c7-a1e7-780ed3ea3614.jpeg)

Reply:
OK - got it. Thanks :+1:

Basically, we don't want `' second'` to be valid. How about we make it `[0-9a-z]+`. rather than `\d*`. That should ensure there is _some value_ there and I think that should catch all the cases we care about.

Reply:
```suggestion
  /[0-9a-z]+ (second|seconds|minute|minutes|hour|hours|day|days|month|months|year|years)/
```

Reply:
Regex modified